src/meetbot/features/record.ts

```diff
@@ -12,37 +11,36 @@ export const attach = (bot: Bot) => {
 
 const startRecording = async (page: Page) => {
 	if (
-		'Rec' ===
-		(await (await page.$('.F9AaL'))?.evaluate((element) => element.textContent))
+		'REC' ===
+		(await (
+			await page.$('.KHSqkf')
```

Comment:
Perhaps not for this PR, but we should move all these selectors to a single file and give them appropriate names, better for maintaining the project, wdyt?

Reply:
https://github.com/balena-io-playground/meetbot/issues/57